envoyauth/request.go

```diff
@@ -160,7 +160,11 @@ func getParsedBody(logEntry *logrus.Entry, headers map[string]string, body strin
 			if !known {
 				return nil, false, nil
 			}
+		} else {
+			logEntry.Infof("content-type: %s parsing not supported", val)
```

Comment:
What about keeping this at debug level ?

Reply:
yeah, i wasn't quite sure where to put this when i did it. On one hand someone supplying a common content type and not having it parsed may warrant something at Info, but for a more obscure type Debug seems fine.  I went with the verbose option to maybe stave off user-confusion.

That being said, my opinion on that isn't strong; i'll happily drop it to debug if you think that's better.